Dockerfile

```
@@ -22,10 +20,12 @@ COPY --from=build /graphhopper/web/target/graphhopper*.jar ./
 
 COPY ./config-example.yml ./
 
+COPY ./graphhopper.sh ./
+
 VOLUME [ "/data" ]
 
 EXPOSE 8989
 
 HEALTHCHECK --interval=5s --timeout=3s CMD curl --fail http://localhost:8989/health || exit 1
 
-ENTRYPOINT [ "java $JAVA_OPTS $TOOL_OPTS -jar *.jar", "server config-example.yml" ]
+ENTRYPOINT [ "graphhopper.sh", "-a web -c config-example.yml --url https://download.geofabrik.de/europe/germany/berlin-latest.osm.pbf" ]
```

```suggestion
ENTRYPOINT [ "graphhopper.sh", "-a", "server", "-c", "config-example.yml" ]
```

For a docker image, I think it would be better not to have a default data source.